astropy/io/ascii/tests/test_html.py

```diff
@@ -603,3 +604,34 @@ def test_raw_html_write_clean():
     <td><em>y</em></td>
    </tr>"""
     assert expected in out.getvalue()
+
+def test_write_table_html():
```

Comment:
You should also test with a table that has masked values where you set `fill_values` to `[(ascii.masked, 'TEST')]` and confirm that the masked table entries are indeed `TEST`.